### PR TITLE
include algolia index cloning

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary

include algollia index promotion

### CODE REVIEW

File: .github/helpers/clone_algolia_index.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+# variables
+# ALGOLIA_API_KEY string
+# ALGOLIA_APP_ID string
+# ALGOLIA_SOURCE_INDEX string
+# ALGOLIA_DEST_INDEX string
+
+curl -X POST \
+     -H "X-Algolia-API-Key: ${API_KEY}" \
```
Comment (AndrewJun97) [inline .github/helpers/clone_algolia_index.sh:9]: Should this be ALGOLIA_API_KEY?

File: .github/helpers/clone_algolia_index.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+     -H "X-Algolia-Application-Id: ${APPLICATION_ID}" \
```
Comment (AndrewJun97) [inline .github/helpers/clone_algolia_index.sh:10]: Should this be ALGOLIA_APP_ID?

File: .github/helpers/clone_algolia_index.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+     --data-binary "{ \"operation\": \"copy\", \"destination\":\"$ALOGOLIA_DEST_INDEX\" }" \
```
Comment (AndrewJun97) [inline .github/helpers/clone_algolia_index.sh:11]: Typo: $ALOGOLIA_DEST_INDEX 

But should it be ${ALGOLIA_DEST_INDEX}?

File: .github/helpers/clone_algolia_index.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    "https://${APPLICATION_ID}.algolia.net/1/indexes/$AGOLIA_SOURCE_INDEX/operation"
```
Comment (AndrewJun97) [inline .github/helpers/clone_algolia_index.sh:12]: Typo: $AGOLIA_SOURCE_INDEX 

but should it be ${ALGOLIA_SOURCE_INDEX}?

File: .github/helpers/delete_algolia_index.sh [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+# variables
+# ALGOLIA_API_KEY string
+# ALGOLIA_APP_ID string
+# ALGOLIA_INDEX_NAME string
+
+curl -X DELETE \
+     -H "X-Algolia-API-Key: ${ALGOLIA_API_KEY}" \
+     -H "X-Algolia-Application-Id: ${ALGOLIA_APP_ID}" \
+    "https://${ALGOLIA_APP_ID}.algolia.net/1/indexes/$ALGOLIA_INDEX_NAME"
```
Comment (AndrewJun97) [inline .github/helpers/delete_algolia_index.sh:10]: Should $ALGOLIA_INDEX_NAME be ${ALGOLIA_INDEX_NAME}?

**Review by AndrewJun97 — APPROVED** [review]

File: .github/helpers/clone_algolia_index.sh (hunk 1 above)
Comment (patricksimonian) [reply .github/helpers/clone_algolia_index.sh:9]: yes it should!

Comment (patricksimonian) [inline .github/helpers/clone_algolia_index.sh:9]:
```suggestion
     -H "X-Algolia-API-Key: ${ALGOLIA_API_KEY}" \
```

File: .github/helpers/clone_algolia_index.sh (hunk 2 above)
Comment (patricksimonian) [inline .github/helpers/clone_algolia_index.sh:10]:
```suggestion
     -H "X-Algolia-Application-Id: ${ALGOLIA_APPLICATION_ID}" \
```

File: .github/helpers/clone_algolia_index.sh (hunk 3 above)
Comment (patricksimonian) [inline .github/helpers/clone_algolia_index.sh:11]:
```suggestion
     --data-binary "{ \"operation\": \"copy\", \"destination\":\"${ALOGOLIA_DEST_INDEX}\" }" \
```

File: .github/helpers/clone_algolia_index.sh (hunk 4 above)
Comment (patricksimonian) [inline .github/helpers/clone_algolia_index.sh:12]:
```suggestion
    "https://${APPLICATION_ID}.algolia.net/1/indexes/${ALGOLIA_SOURCE_INDEX}/operation"
```

File: .github/helpers/delete_algolia_index.sh (hunk 5 above)
Comment (patricksimonian) [inline .github/helpers/delete_algolia_index.sh:10]:
```suggestion
    "https://${ALGOLIA_APP_ID}.algolia.net/1/indexes/${ALGOLIA_INDEX_NAME}"
```

**Pull request merged**